### DISCUSSION TITLE
cjkuni-uming-fonts on RockyLinux 8.5

### DISCUSSION THREAD

**Discussion by vieira-temes** [rustdesk/rustdesk#1859] [General]
I'm running RockyLinux 8.5 and trying to install the RustDesk client.

Is there any way to skip the installation of cjkuni-uming-fonts? I don't need them.

If not, I see that I need to install the Raven repository, but can't find how. Can you please post a simple command line that installs that repo?

Thanks for any help.

**Comment by vieira-temes** [comment]
I found the repository rpm here:

https://pkgs.dyn.su/el8/base/x86_64/raven-release.el8.noarch.rpm

I just downloaded it and installed it via

`dnf install raven-release.el8.noarch.rpm.`

Problem solved.

Still, I'd like to see cjkuni-umin-fonts removed as a hard requirement.

**Comment by rustdesk** [comment]
Thanks, we will remove it in the future.

**Reply by albgen** [reply]
just tried on RockyLinux and this remains as issue

**Reply by mjfrazer** [reply]
Still an issue on Alma 9 as well

**Reply by mjfrazer** [reply]
To install on alma 9:
```
  sudo yum install python3-wheel python3-devel python3-pip
  sudo yum install https://pkgs.dyn.su/el8/base/x86_64/cjkuni-uming-fonts-0.2.20080216.1-62.el8.noarch.rpm
  sudo yum install https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-fedora28-centos8.rpm
```
